pkg/cli/image/archive/archive.go

```diff
@@ -392,20 +392,23 @@ func createTarFile(path, extractDir string, hdr *tar.Header, reader io.Reader, L
 		}
 	}
 
-	var errors []string
 	for key, value := range hdr.Xattrs {
 		// exclude security.capability unless you're root, Lsetxattr only works
 		// for linux based systems so that's fine
 		if key == "security.capability" && currentUser != nil && currentUser.Username != "root" {
 			continue
 		}
 		if err := system.Lsetxattr(path, key, []byte(value), 0); err != nil {
+			if err == system.ErrNotSupportedPlatform {
```

Comment:
Maybe we should at least at some kind of warning or `klog.V(1)` here, in case someone wants to see it. Same for the error below.